### DISCUSSION TITLE
The filter 9gag in blocked services doesn't work.

### DISCUSSION THREAD

**Discussion by MallemIsmail** [AdguardTeam/AdGuardHome#3564] [General]
Trying to block 9gag from Blocked services it doesn't take effect.

**Comment by ainar-g** [comment]
Thanks for the report!  It seems like there has been an error in one of our filter lists.  We have fixed it, and it should be shipped to the `edge` channel in a couple of hours as well as to the `beta` channel next Tuesday.
